packages/geospatial/src/processing/raster-vector-fusion.ts

```
@@ -0,0 +1,670 @@
+    this.emit('progress', 0, 'Starting fusion operation');
+
+    let outputFeatures: ExtractedFeature[] = [];
+    const zonalStats: Record<string, TileStatistics> = {};
+
+    try {
+      switch (this.config.fusionMethod) {
+        case 'overlay':
+          outputFeatures = await this.overlayFusion(scene, vectorFeatures, rasterData);
+          break;
+        case 'zonal_stats':
+          const { features, stats } = await this.zonalStatsFusion(scene, vectorFeatures, rasterData);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)